tests/test_data/test_sinan_fetch.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+from pathlib import Path
+
+import pandas as pd
+from pysus.online_data.SINAN import download
+
+from epigraphhub.data.sinan_fetch import (
+    engine,
+    parquet_to_df,
+    save_to_pgsql,
+    schema_from_df,
+)
+
+
+class TestFethSinan(unittest.TestCase):
+    def setUp(self):
+        self.engine = engine
+        self.fname = "ZIKABR17.parquet"
+        self.fn = download(2017, "zika", return_fname=True)
+        self.fname_pq = Path(f"./{self.fname}/")
+
+    def test_schema_from_df(self):
+        table_name = schema_from_df(self.fname)[0]
+        query_psql = schema_from_df(self.fname)[1]
+        parquet_to_df = schema_from_df(self.fname)[2]
+
+        self.assertEqual(table_name, '"brasil"."zikabr17"')
+        self.assertIn("CREATE TABLE brasil.zikabr17", query_psql)
+        self.assertIsInstance(parquet_to_df, pd.DataFrame)
+        self.assertTrue(self.fname_pq.exists())
+
+    def test_parquet_to_df(self):
+        df_compare = parquet_to_df(self.fname)
+        self.assertEqual(df_compare.shape[0], 32684)
```

Comment:
The size of this file can change, right? What could break the test since the number of rows is hard coded

Reply:
That's right, the dataframe size will never change. The same parquet will always be downloaded: *self.fname = "ZIKABR17.parquet*, for this case if the result is not the same as **32684**, something was not successful in concatenating the dataframes.

Reply:
Ok. I was afraid that the dataframe size could change with some later updates.

Reply:
> Ok. I was afraid that the dataframe size could change with some later updates.

If I'm not mistaken, the *PRELIMINAR* datasus data may change, but these are datasets after 2019.